cmd/get.go

```diff
@@ -92,7 +92,15 @@ func (o *GetOptions) Complete(f Factory, args []string) (err error) {
 			args = args[1:]
 		}
 	}
-	o.Refs = args
+	o.Refs = make([]string, 0, len(args))
```

Comment:
this has a funny side effect if no args are provided, which, for `qri use`, is kinda the point 😄

Reply:
Yeah, this is a bug, GetDatasetRefString is never called in that case. I'll fix.